### PR TITLE
[ISSUE #769]Service input box add serviceVersion and serviceGroup

### PR DESCRIPTION
**Pull request by haoyann**
The branch of 0.2.0 is master-0.2.0. If your PR is to solve the problem of 0.2.0, please submit the PR to it.
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Fro #769.

## Brief changelog

- Service governance list input box add `serviceVersion` and `serviceGroup`.
- Service governance list query return `id` used for CRUD.

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist.
* [ ] Run `mvn clean compile --batch-mode -DskipTests=false -Dcheckstyle.skip=false -Drat.skip=false -Dmaven.javadoc.skip=true to make sure basic checks pass.

### CODE REVIEW

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/RouteServiceImpl.java [hunk 1]
```diff
@@ -180,7 +177,12 @@ public AccessDTO findAccess(String id) {
         if (config != null) {
             RoutingRule ruleDTO = YamlParser.loadObject(config, RoutingRule.class);
             List<String> blackWhiteList = RouteUtils.filterBlackWhiteListFromConditions(ruleDTO.getConditions());
-            return RouteUtils.convertToAccessDTO(blackWhiteList, ruleDTO.getScope(), ruleDTO.getKey());
+            AccessDTO accessDTO = RouteUtils.convertToAccessDTO(blackWhiteList, ruleDTO.getScope(), ruleDTO.getKey());
+            accessDTO.setId(id);
+            if (ruleDTO.getScope().equals(Constants.SERVICE)) {
```
Comment (horizonzy) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/RouteServiceImpl.java:182]: Constants.SERVICE.equals(ruleDTO.getScope())

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/OverrideServiceImpl.java [hunk 2]
```diff
@@ -426,6 +437,10 @@ public BalancingDTO findBalance(String id) {
                 for (OverrideConfig overrideConfig : configs) {
                     if (Constants.BALANCING.equals(overrideConfig.getType())) {
                         BalancingDTO balancingDTO = OverrideUtils.configtoBalancingDTO(overrideConfig, overrideDTO.getScope(), id);
+                        balancingDTO.setId(id);
+                        if (overrideDTO.getScope().equals(Constants.SERVICE)) {
```
Comment (horizonzy) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/OverrideServiceImpl.java:441]: Constants.SERVICE.equals(overrideDTO.getScope())

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/OverrideServiceImpl.java [hunk 3]
```diff
@@ -213,7 +213,14 @@ public DynamicConfigDTO findOverride(String id) {
         String config = dynamicConfiguration.getConfig(path);
         if (config != null) {
             OverrideDTO overrideDTO = YamlParser.loadObject(config, OverrideDTO.class);
-            return OverrideUtils.createFromOverride(overrideDTO);
+            DynamicConfigDTO dynamicConfigDTO = OverrideUtils.createFromOverride(overrideDTO);
+            if (dynamicConfigDTO != null) {
+                dynamicConfigDTO.setId(id);
+                if (overrideDTO.getScope().equals(Constants.SERVICE)) {
```
Comment (horizonzy) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/OverrideServiceImpl.java:219]: also equals order problem.

Comment (haoyann) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/impl/OverrideServiceImpl.java]: done.

**Review by KeRan213539 — APPROVED** [review]

**Pull request merged**